espnet/asr/pytorch_backend/asr_mix.py

```diff
@@ -83,6 +83,7 @@ def __call__(self, batch, device):
         # batch should be located in list
         assert len(batch) == 1
         xs, ys = batch[0]
+        ys = list(ys)  # Convert zip object to list in python 3.x
```

Comment:
Could you make sure that it's working on both 3.x and 2.x?
(for now, we try to support both versions).

Reply:
@simpleoier, could you answer this question? This is just for the confirmation.

Reply:
Yes. I tested on both Python 3.x and 2.x, it works well. Besides, Marc's experience also confirmed that it works on Python 3.x.